helix-term/src/commands.rs

```diff
@@ -417,7 +417,8 @@ impl MappableCommand {
         decrement, "Decrement",
         record_macro, "Record macro",
         replay_macro, "Replay macro",
-        command_palette, "Open command palette",
+        command_palette, "Open command pallete",
+        toggle_syntax, "Toggle (on/off) syntax for current document",
```

Comment:
Instead of doing it this way, I think it can be done through `:set-option toggle-syntax on/off`. That way we don't have to prompt.